trin-core/src/jsonrpc/handlers.rs

```diff
@@ -22,44 +24,56 @@ pub struct JsonRpcHandler {
 
 impl JsonRpcHandler {
     pub async fn process_jsonrpc_requests(mut self) {
-        while let Some(request) = self.portal_jsonrpc_rx.recv().await {
-            let response: Value = match request.endpoint {
+        while let Some(portal_jsonrpc_req) = self.portal_jsonrpc_rx.recv().await {
+            let response: Value = match portal_jsonrpc_req.endpoint {
                 TrinEndpoint::Discv5Endpoint(endpoint) => match endpoint {
                     Discv5Endpoint::NodeInfo => self.discovery.read().await.node_info(),
                     Discv5Endpoint::RoutingTableInfo => {
                         self.discovery.write().await.routing_table_info()
                     }
+                    Discv5Endpoint::FindNodes => {
+                        // TODO: Add RPC handler for every endpoint and validate parameters
+                        Value::String("Disscv5 FindNodes endpoint not implemented yet!".to_owned())
```

Comment:
Couldn't we just call `discovery.find_node(..)` here? Also minor typo `Disscv5`. Also, unless I'm missing something... it does seem like this endpoint is implemented, why are we returning this message?

Reply:
I left this as a placeholder because we still don't have validations for the parameters, but probably we can still call discovery.find_nodes anyways.